### DISCUSSION TITLE
3 different links in a markdown note are generated as 1 href link

### DISCUSSION THREAD

**Discussion by ooker777** [ObsidianPublisher/obsidian-github-publisher#142] [General]
When the file [index.md](https://github.com/ooker777/mkdocs-hyperlink-bug/blob/main/docs/index.md) is generated to be the file [index.html](https://github.com/ooker777/mkdocs-hyperlink-bug/blob/main/site/index.html), 3 different links to 3 different files are generated as 1 same href link.

Here are the original links:
```
[[./📜 Tài nguyên/💎 Giới thiệu về Obsidian/index|💎 Giới thiệu về Obsidian]]
[[./⚔️ Lớp Obsidian và Git/index|⚔️ Lớp Obsidian và Git]]
[[./⚡Hiểu biết sâu/index|⚡Hiểu biết sâu]]
```

They are generated as:
```html
<a href="%F0%9F%93%9C%20T%C3%A0i%20nguy%C3%AAn/%F0%9F%92%8E%20Gi%E1%BB%9Bi%20thi%E1%BB%87u%20v%E1%BB%81%20Obsidian/Theo%20t%C3%ADnh%20n%C4%83ng%20c%E1%BB%A7a%20plugin/" title="💎 Giới thiệu về Obsidian">💎 Giới thiệu về Obsidian</a>
<a href="%F0%9F%93%9C%20T%C3%A0i%20nguy%C3%AAn/%F0%9F%92%8E%20Gi%E1%BB%9Bi%20thi%E1%BB%87u%20v%E1%BB%81%20Obsidian/Theo%20t%C3%ADnh%20n%C4%83ng%20c%E1%BB%A7a%20plugin/" title="⚔️ Lớp Obsidian và Git">⚔️ Lớp Obsidian và Git</a>
<a href="%F0%9F%93%9C%20T%C3%A0i%20nguy%C3%AAn/%F0%9F%92%8E%20Gi%E1%BB%9Bi%20thi%E1%BB%87u%20v%E1%BB%81%20Obsidian/Theo%20t%C3%ADnh%20n%C4%83ng%20c%E1%BB%A7a%20plugin/" title="⚡Hiểu biết sâu">⚡Hiểu biết sâu</a>
```

Notice the titles are different.

Why is this? Other pages seems fine. Here is the [mkdocs.yml](https://github.com/ooker777/mkdocs-hyperlink-bug/blob/main/mkdocs.yml), but I don't think it has any problem in it.

I guess this is not a problem of the plugin, but perhaps you have some insights about it.

**Comment by Mara-Li** [comment]
It's not a issue with Obsidian, Obsidian Github Publisher or my template, but an issue of Mkdocs or a plugin.
Please, check your configuration and create appropriate issue on repository.
